mainnet-1/specs/movement.json

```diff
@@ -0,0 +1,632 @@
+{
+    "proposal": {
+        "title": "Add Specs: Movement",
+        "description": "Adding new specification support for relaying Movement data on Lava",
+        "specs": [
+            {
+                "index": "MOVEMENT",
+                "name": "movement mainnet",
+                "enabled": true,
+                "imports": [
+                    "APT1"
+                ],
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 5,
+                "blocks_in_finalization_proof": 5,
+                "average_block_time": 1000,
+                "allowed_block_lag_for_qos_sync": 50,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "5000000000"
+                },
+                "api_collections": [
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "rest",
+                            "internal_path": "",
+                            "type": "GET",
+                            "add_on": ""
+                        },
+                        "apis": [
+                            {
+                                "name": "/",
```

Comment:
If you added inheritance why we still have duplicate APIs. 
please take a look at the movement spec and make sure to understand how inheritance works. 
https://github.com/lavanet/lava/blob/main/cookbook/specs/movement.json

Reply:
i will fix it